syz-hub/state/state.go

```diff
@@ -188,30 +192,35 @@ func (st *State) pendingInputs(mgr *Manager) ([][]byte, int, error) {
 	if len(records) > maxRecords {
 		sort.Sort(recordSeqSorter(records))
 		pos := maxRecords
-		maxSeq = records[pos].Seq
-		for pos+1 < len(records) && records[pos+1].Seq == maxSeq {
+		maxSeq = getSeq(records[pos].Seq)
```

Comment:
`getSeq` is easy to forget in one place, and ordering of sequences is important
I think it's better to move this directly to db